### PR TITLE
add get token info helper func

### PR DESCRIPTION
**Pull request by Tianhao-Gu**

### CODE REVIEW

**Comment by Tianhao-Gu** [comment]
@briehl Do you mind taking a look?

File: test/authSpec.js [hunk 1]
```diff
@@ -3,6 +3,8 @@
 var expect = require('chai').expect;
 const Auth = require('../lib/index.js').Auth;
 const jsdom = require('jsdom');
+const auth_token = '6CQKPLID5HLKKURMX7NYPORJ6J3T3Z23';
```
Comment (briehl) [inline test/authSpec.js]: don't check your tokens in! :P

File: src/auth.ts [hunk 2]
```diff
@@ -43,4 +75,76 @@ export class Auth {
...
+            method: callParams.method,
+            headers: {
+                'Content-type': 'application/json',
+                'Authorization': token
+            }
+        };
+
+        return fetch(call_url, request)
+            .then(this.handleFetchErrors)
+            .then(response => response.json())
```
Comment (briehl) [inline src/auth.ts]: missing semicolon

File: test/authSpec.js (hunk 1 above)
Comment (Tianhao-Gu) [reply test/authSpec.js]: 👍

File: src/auth.ts (hunk 2 above)
Comment (Tianhao-Gu) [reply src/auth.ts]: 👍

**Comment by Tianhao-Gu** [comment]
@briehl Thanks for review. Addressed your comments.

**Pull request merged**